src/components/MessageWindow.jsx

```diff
@@ -0,0 +1,65 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import { useState, useEffect, useRef } from 'react';
+import MessageList from './MessageList';
+import MessageInput from './MessageInput';
+import chatData from '/public/chatData.json';
+import userData from '/public/userData.json';
+
+const MessageWindow = () => {
+  // 메세지 입력창을 포함한 모든 대화창
+
+  const [messages, setMessages] = useState([]);
+  const [users] = useState(userData);
+
+  // 컴포넌트가 처음 렌더링될 때 JSON 데이터 불러오기
+  useEffect(() => {
+    setMessages(chatData); // 초기 메세지 로드
+  }, []);
```

Comment:
json 데이터를 이렇게 바로 초기화에 사용할 수 있습니다!
```suggestion
  const [messages, setMessages] = useState(chatData);
  const [users] = useState(userData);
```